src/components/ViewerSideNav/ViewerSideNav.style.ts

```diff
@@ -84,13 +96,13 @@ export const SidebarNavLink = styled(Link, { shouldForwardProp: isPropValid })<S
   position: relative;
   align-items: center;
   &:hover {
-    background-color: rgba(0, 0, 0, 0.12);
+    background-color: ${({ isStudio }) => (isStudio ? colors.transparentWhite[4] : 'rgba(0,0,0,0.12)')};
   }
   &:focus {
-    background-color: rgba(0, 0, 0, 0.24);
+    background-color: ${({ isStudio }) => (isStudio ? colors.transparentWhite[6] : 'rgba(0,0,0,0.24)')};
   }
   &:active {
-    background-color: rgba(0, 0, 0, 0.4);
+    background-color: ${({ isStudio }) => (isStudio ? colors.transparentWhite[32] : 'rgba(0,0,0,0.4)')};
   }
```

Comment:
I think we should use some of the colors from `transparentBlack` but not sure if possible for all of them